### PR TITLE
Update associacao-brasileira-de-normas-tecnicas-ufrgs.csl

### PR DESCRIPTION
**Pull request by PriscilaSJUFRGS**
Style update to implement standard's changes

### CODE REVIEW

**Comment by RENANZG** [comment]
Hello, very nice version, but it's not working.
Olá, muito legal a versão, mas não está funcionando.

For example, this is a complete style model:
Por exemplo, este é um modelo de estilo completo:
**Associação Brasileira de Normas Técnicas (note, Português - Brasil) (2021-10-07 02:42:26)** [https://www.zotero.org/styles/associacao-brasileira-de-normas-tecnicas-note](https://www.zotero.org/styles/associacao-brasileira-de-normas-tecnicas-note)

You can see that there is an advanced option for the footnote format.
É possível ver que há uma opção avançada para o formato de nota de rodapé.

screen-2023-08-27-16-37-44

If I test it in practice by selecting the extension in the text editor, the citation in the footnote looks like this:  
Testando na prática, selecionando na extensão do editor de texto, a citação na nota de rodapé fica assim:  

> ¹ FOUCAULT, Michel, **Vigiar e punir: nascimento da prisão**, 39. ed. Petropolis: Vozes, 2011.

(*em princípio este negrito estaria errado, mas tem a questão da [impossibilidade do CSV](https://github.com/citation-style-language/schema/pull/203) ?! )

With your version in the footnote (author date, not complete):
Com a sua versão na nota de rodapé (autor data e não completa):

> ¹ (Foucault, 2011)

Thanks !
Grato !

IMPORTANT REFERENCES:
https://github.com/AAguiarCAM/csl-abnt/tree/master
https://forums.zotero.org/discussion/106403/estilo-abnt-10520-2023/p1
_______________________________________________________________________
I've been helping with the Translator ISBN that's about to be released:
Eu tive ajudando no Translator ISBN que vai ser lançado:

The download location, in the right-hand corner "download raw file": 
O local de download, no canto direito "download raw file": https://github.com/zotero/translators/blob/d8183fe0e17830e327bc1cd8a2057d985de84532/Camara%20Brasileira%20do%20Livro%20ISBN.js


The place to put it is in the translator folder as it says "Zotero translators are stored as individual JavaScript files in the "translators" subdirectory of the Zotero data directory." as it says here for developers  https://www.zotero.org/support/dev/translators  
O local para colocar é na pasta translator como diz "Zotero translators are stored as individual JavaScript files in the “translators” subdirectory of the Zotero data directory." como diz aqui para desenvolvedores https://www.zotero.org/support/dev/translators  

_______________________________________________________________________

**Comment by PriscilaSJUFRGS** [comment]
> 
Thanks for your feedback, @RENANZG 

We have a specific style for footnotes, that's why this one will not have that option. If that is the only error that you found, it's not an error, but a choice.

About subtitles, yes, it's because of csl limitation.

**Comment by adam3smith** [comment]
What they're saying is that this is the author-date version of the style, so (Foucault, 2011) is what you should expect. There's a separate note version, currently not updated from ABNT 2023, for UFRG: https://www.zotero.org/styles?q=id%3Aassociacao-brasileira-de-normas-tecnicas-ufrgs-note-initials-with-ibid

**Comment by PriscilaSJUFRGS** [comment]
Exactly, @adam3smith . Thank you!

@RENANZG, we are waiting the merge of this style to be complete so we can advance on updating the other ones.

**Comment by PriscilaSJUFRGS** [comment]
@adam3smith, do you know why this update is taking so long to be merged?

**Comment by PriscilaSJUFRGS** [comment]
Did we do something wrong? 😭 I see many other styles beeing merged and our is still here...

File: associacao-brasileira-de-normas-tecnicas-ufrgs.csl [hunk 1]
```diff
@@ -226,20 +227,20 @@
     <!-- Macro responsável por mostrar o autor na citação -->
     <choose>
       <if type="broadcast motion_picture song" match="any">
-        <text variable="title" form="short" text-case="uppercase" quotes="false"/>
+        <text variable="title" form="short" text-case="capitalize-first" quotes="false"/>
```
Comment (adam3smith) [inline associacao-brasileira-de-normas-tecnicas-ufrgs.csl:230]: Is there any reason you're using 'capitalize-first' for all the titles and author names? In 99% of the cases, these will be properly stored in the database, including in the rare cases where authors strongly prefer their names to be lowercased (danah boyd, e.g.)
I'd suggest removing the text-case attribute entirely throughout.

File: associacao-brasileira-de-normas-tecnicas-ufrgs.csl [hunk 2]
```diff
@@ -691,15 +687,15 @@
...
-          <!-- Documento (Zotero) e Programa de computador (Mendeley) -->
+          <!-- Preprint (Zotero) e Programa de computador (Mendeley) -->
           <group>
             <text macro="author" suffix=". "/>
             <text variable="authority" suffix=". "/>
             <text macro="title" suffix=" "/>
             <text variable="medium" suffix=". "/>
-            <!--incluído para ter a versão de programa de computador no mendeley-->
+            <!-- incluído para ter a versão de programa de computador no mendeley -->
             <text variable="number" prefix="Versão " suffix=". "/>
```
Comment (adam3smith) [inline associacao-brasileira-de-normas-tecnicas-ufrgs.csl:697]: That's doesn't work anymore -- number gets (properly) mapped to archive ID (e.g. arXiv ID) for preprints in Zotero. You can't use it for "Version" for `article`, sorry.

**Review by adam3smith — CHANGES_REQUESTED** [review]
Two minor issues, please fix.
Beyond that, I do really wish you'd rethink your reliance on affixes/suffix instead of group delimiters for punctuation: it makes styles much more prone to incorrect output with missing data, but I won't insist here.

File: associacao-brasileira-de-normas-tecnicas-ufrgs.csl (hunk 2 above)
Comment (PriscilaSJUFRGS) [reply associacao-brasileira-de-normas-tecnicas-ufrgs.csl]: Thanks, @adam3smith ! variable excluded.

File: associacao-brasileira-de-normas-tecnicas-ufrgs.csl (hunk 1 above)
Comment (PriscilaSJUFRGS) [reply associacao-brasileira-de-normas-tecnicas-ufrgs.csl]: Thanks, @adam3smith ! Attribute removed.

**Comment by PriscilaSJUFRGS** [comment]
Thank you for your reply, @adam3smith ! We don't know how to use github very well and still learning about csl. We'll try to make it better our use of prefix/suffix. And thanks for always helping us!

File: associacao-brasileira-de-normas-tecnicas-ufrgs.csl (hunk 1 above)
Comment (PriscilaSJUFRGS) [reply associacao-brasileira-de-normas-tecnicas-ufrgs.csl]: Actually, I'll have to make some changes on this issue, but I don't know how to do it.

How can I make a `name-part name="family"` and a substitute clause when there is no author and we use title be uppercase only at bibliography?

```
			<else-if type="entry-dictionary entry-encyclopedia" match="any">
				<names variable="author">
					<!-- para outros tipos de documentos mantém a posição original -->
					<name delimiter="; " delimiter-precedes-et-al="never" delimiter-precedes-last="always" name-as-sort-order="all">
						<name-part name="family"/>
						<name-part name="given"/>
					</name>
					<et-al font-style="italic"/>
					<label form="short" text-case="lowercase" prefix=" (" suffix=".)"/>
					<substitute>
						<!-- na ausência de um autor, utiliza organizador, editor, tradutor ou título  -->
						<text variable="title" />
					</substitute>
				</names>
			</else-if>
```

**Comment by adam3smith** [comment]
Thanks & sorry for the slow review

**Comment by weslleymberg** [comment]
Hello, @PriscilaSJUFRGS 
After the update, the location field for Books is showing up between the name of the authors and the title.

Example in the image below:
image

**Comment by PriscilaSJUFRGS** [comment]
> Hello, @PriscilaSJUFRGS After the update, the location field for Books is showing up between the name of the authors and the title.
> 
> Example in the image below: image
> 
> Thank you for your work on this style!

Thank you, @weslleymberg ! We made some corrections and now waiting revision.

**Pull request merged**